Client/Frontend/Home/ActivityStreamPanel.swift

```diff
@@ -452,6 +452,9 @@ extension ActivityStreamPanel {
     }
 }
 
+// TODO Not sure where this should go
+let PocketLocaleIdentifier = ["en_US", "en_GB", "en_ZA", "en_US", "en_GB", "en_ZA", "de_DE", "de_AT", "de_CH"]
```

Comment:
I think having this and all the locale logic in the PocketFeedAPI class would be better. 
ASPanel is starting to get pretty big